src/Package/Impl/DataInspect/VariableView.xaml.cs

```diff
@@ -76,8 +74,8 @@ public partial class VariableView : UserControl, IDisposable {
 
             var currentItem = EnvironmentComboBox.SelectedItem as REnvironment;
             if (currentItem != null && !e.Environments[0].FrameIndex.HasValue) {
-                int index = e.Environments.FindIndex((env) => env.Name == currentItem.Name);
-                selectedIndex = index == -1 ? selectedIndex : index;
+                var indexes = e.Environments.IndexWhere((env) => env.Name == currentItem.Name);
+                selectedIndex = indexes.Count() > 0 ? indexes.First() : selectedIndex;
```

Comment:
Use `Any` rather than `Count` to avoid unnecessarily enumerating the whole thing. Or better yet:
```
selected Index = ...IndexWhere(...).FirstOrDefault() ?? selectedIndex
```